backend/Wheelingful.Data/Extensions/SeedExtension.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Wheelingful.Core.Enums;
+
+namespace Wheelingful.Data.Extensions;
+
+internal static class SeedExtension
+{
+    public static void SeedRoles(this ModelBuilder builder)
+    {
+
+        builder.Entity<IdentityRole>().HasData(
+            new IdentityRole
+            {
+                Id = "1",
+                Name = nameof(UserRoleEnum.Admin),
+                NormalizedName = nameof(UserRoleEnum.Admin).ToUpper()
+            },
+            new IdentityRole
+            {
+                Id = "2",
+                Name = nameof(UserRoleEnum.Reader),
+                NormalizedName = nameof(UserRoleEnum.Reader).ToUpper()
+            },
+            new IdentityRole
+            {
+                Id = "3",
+                Name = nameof(UserRoleEnum.Author),
+                NormalizedName = nameof(UserRoleEnum.Author).ToUpper()
+            }
+        );
+    }
+
+    public static void SeedUsers(this ModelBuilder builder)
+    {
+        var passwordHasher = new PasswordHasher<IdentityUser>();
+
+        builder.Entity<IdentityUser>().HasData(
+            new IdentityUser
+            {
+                Id = "1",
+                UserName = "admin@wheelingful.com",
+                NormalizedUserName = "ADMIN@WHEELINGFUL.COM",
+                Email = "admin@wheelingful.com",
+                NormalizedEmail = "ADMIN@WHEELINGFUL.COM",
+                EmailConfirmed = true,
+                PasswordHash = passwordHasher.HashPassword(null, "#SYSADMIN123"),
```

Comment:
have you solved the issue with a new hash generated on each transaction?

Reply:
No, I haven't. It still creates a new hash on a new migration. For now it is faster to delete the lines each time than to come up with a solution. But if you insist...